### ISSUE TITLE
Accessibility: UI elements should respond to click events, not just touch ones

### ISSUE THREAD

**Issue by vick08** [ToastWallet/core#14]
I tried Toast Wallet on Android, iOS and Mac, and in all three cases observed the same accessibility barrier (coming from the shortcomings of the JS/CSS implementation). In particular, when a screen reader, e.g. Talkback on Android, VoiceOver on Mac OS/iOS, attempts to activate any of the UI elements, nothing happens. Quite likely, the reason is that those elements expect touch/mouse events exclusively. For accessibility clients to function properly, the click events should be respected as well.
As it stands now, many disabled users who rely on accessibility tools will have difficult time using Toast Wallet app(s).

**Comment by ToastWallet** [comment]
Hi vick, thanks for your report. We do have a click proxy https://github.com/ToastWallet/core/blob/master/www/index.html#L1614 this code should be translating traditional clicks on PC to the touch events that Toast uses. I think it's just a case of debugging this code to figure out why accessibility programs are not firing the event.

We would be most grateful if you could play around with it and see if you can come up with a modification that would get it working on your accessibility software. Thanks!
